OCABundles/credentials/ongov-digital-trust/ontario-test-credential/OCABundle.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "capture_base": {
+      "attributes": {
```

Comment:
The attributes list does not match the attributes in the schema that is identified.  Expecting (based on the schema: 

          "region",
          "issued",
          "street_address",
          "birthdate",
          "family_name",
          "country",
          "postal_code",
          "locality",
          "given_names"